### ISSUE TITLE
Fujitsu 6x4 Digit Version Python Code Available

### ISSUE THREAD

**Issue by 7enderhead** [bacher09/pwgen-for-bios#221]
I researched the adapted version of dogbert's code for Fujitsu 6x4 digit hash passwords and [adapted it to run with Python 3](https://gist.github.com/7enderhead/66e710b3d60f8241a997d8a3ff7b337c) . I have tested the result on a Celsius H730 laptop. Please feel free to incorporate the code on your web page.

**Comment by gstamosbg** [comment]
Do you have anything for 6x4 hexadecimal?

**Comment by 7enderhead** [comment]
No, I haven't. What would you need it for? As far as I know, the master password hash is given in decimal form anyway.
